requirements.txt

```diff
@@ -21,7 +21,7 @@ eth-hash[pycryptodome]==0.1.4
 pytest-timeout==1.3.3
 cachetools==3.1.1
 websockets==8.0.2
-
+grpc==1.26.0
```

Comment:
seems to be `grpcio`

Reply:
![image](https://user-images.githubusercontent.com/38544039/73703523-fd82a580-46a4-11ea-9992-4a554caf14c0.png)
https://pypi.org/project/grpcio/